subprojects/logging/src/main/java/org/gradle/internal/logging/console/WorkInProgressRenderer.java

```diff
@@ -183,6 +222,20 @@ private boolean isChildAssociationAlreadyExists(OperationIdentifier parentId) {
         if (children != null && !children.isEmpty()) {
             return true;
         }
+
+        return false;
+    }
+
+    // Any ProgressOperation in the parent chain has a message, the operation is considered renderable.
+    private boolean isRenderable(ProgressOperation operation) {
+        for (ProgressOperation current = operation;
+             current != null && !"org.gradle.internal.progress.BuildProgressLogger".equals(current.getCategory());
```

Comment:
:o: Nitpick: extract a constant here. 

Aside, I had refactored this constant out of `BuildStatusRenderer` originally but put it back because it was not related to log grouping.